src/components/PageHeader/MeasureActionCenter/MeasureActionCenter.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+
+import { SpeedDial, SpeedDialAction } from "@mui/material";
+import { DeleteOutlined as DeleteIcon } from "@mui/icons-material";
+
+const MeasureActionCenter = () => {
+  const [open, setOpen] = useState(false);
+
+  const actions = [{ icon: <DeleteIcon />, name: "Delete Measure" }];
+
+  return (
+    <div
+      data-testid="action-center"
+      style={{
+        display: "flex",
+        alignItems: "center",
+        height: 40,
+        backgroundColor: open ? "white" : "transparent",
+        borderRadius: 25,
+      }}
+    >
+      <SpeedDial
+        ariaLabel="SpeedDial for actions"
```

Comment:
could be `Measure action center` or something that describes the purpose of this action center